api/src/main/java/org/apache/iceberg/PartitionKey.java

```diff
@@ -28,11 +28,10 @@
 
 /**
  * A struct of partition values.
- * <p>
+ * <br>
  * Instances of this class can produce partition values from a data row passed to {@link #partition(StructLike)}.
  */
 public class PartitionKey implements StructLike, Serializable {
-
```

Comment:
Can you revert unnecessary whitespace changes?

Reply:
Done